.rubocop.yml

```diff
@@ -1200,6 +1200,7 @@ RSpec/DescribeClass:
     - "**/tasks/**/*"
 
 RSpec/EmptyExampleGroup:
+  Enabled: false
```

Comment:
why disable?

Reply:
It's because Rubocop is failing with the empty tests. For example: `spec/services/decidim/challenges/challenge_search_spec.rb:129` because is empty. 

What do you propose?